Pod/Classes/AppViewController.swift

```diff
@@ -25,6 +25,14 @@ import UIKit
 
 open class AppViewController: UIViewController {
     
+    open override var shouldAutorotate: Bool {
+        return installedViewController?.shouldAutorotate ?? true
+    }
+    
+    open override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return installedViewController?.supportedInterfaceOrientations ?? .all
```

Comment:
Similar to above, maybe instead of defaulting to `.all` we should default to `super. supportedInterfaceOrientations `.